components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/DefaultRequestPathBasedSequenceHandler.java

```diff
@@ -222,31 +225,50 @@ protected void handlePostAuthentication(HttpServletRequest request,
     protected String getServiceProviderMappedUserRoles(SequenceConfig sequenceConfig,
                                                        List<String> locallyMappedUserRoles) throws FrameworkException {
 
-        if (locallyMappedUserRoles != null && !locallyMappedUserRoles.isEmpty()) {
-
-            Map<String, String> localToSpRoleMapping = sequenceConfig.getApplicationConfig()
-                    .getRoleMappings();
-
-            boolean roleMappingDefined = false;
+        if (log.isDebugEnabled()) {
+            AuthenticatedUser authenticatedUser = sequenceConfig.getAuthenticatedUser();
+            String serviceProvider = sequenceConfig.getApplicationConfig().getApplicationName();
+            log.debug("Getting Service Provider mapped roles of application: " + serviceProvider +
+                    " of user: " + authenticatedUser);
+        }
 
+        // comma separated SP role mapped role values
```

Comment:
The line comments should be in the standard way right.. 
eg. // This is a comment.

Reply:
Yes. Need to start with a Capital letter and end with a fullstop.